docs/dev/json-api.md

```diff
@@ -66,6 +78,63 @@ Second, we create the `wfo_pdf_upload` entity itself and link the PDF `id`:
 
 The response should also be a 201 with JSON information about the newly created `wfo_pdf_upload` entity.
 
+# Weather story image upload example
+
+Similarly, uploading weather stories is a two-step process. We upload to different endpoints with different field names but the process is the same. First, start with uploading the image(s):
+
+The `field_fullimage` is required.
+
+    curl -sL \
```

Comment:
Yes it definitely _will_ cause problems and we expect them. They have sent information with images that are 10MB and we are hoping to work out a way to get them to send images that are much smaller in byte size

Reply:
We bumped up to 3 MB specifically because (after an analysis of a weeks' worth of weather stories) 18-20% of weather story images are between 2 and 3 MB. In the end, I believe we accept 95% of all images that are sent our way. That's the best compromise I could find -- I definitely don't want to accept 10MB images. (These images are so big because they are 1080p images generated from Google Slides, essentially)

We are gently pushing them to improve these image file sizes but it might take a while before they do it. So that's the situation we are in right now.